style/ruby/README.md

```diff
@@ -0,0 +1,14 @@
+# Ruby Style Enforcement
+
+It is recommended that you use a ruby style enforcement tool such as [RuboCop](/bbatsov/rubocop). This can be integrated in your CI build and optionally use a service like HoundCI to comment on Pull Requests indicating violations.
+
+There is a RuboCop configuration file available in this directory that can be used as a base for a new project.  There are various modifications in that configuration that we have collaboratively determined are acceptable in DLSS.
```

Comment:
We should probably figure out a distribution pattern that uses the rubocop inheritance model. It'd be nice if we could maintain a common style guide here, and let projects inherit and tweak it as appropriate, while still making it easy to update to the latest common style in the future (looking at rubocop, it doesn't look like it supports pulling from remote sources or anything...)